src/containers/BookForm.js

```diff
@@ -6,23 +6,24 @@ import { createBook } from '../actions/index';
 class BookForm extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { id: Math.floor(Math.random() * 100), title: '', category: '' };
-    this.updateTitle = this.updateTitle.bind(this);
-    this.updateCategory = this.updateCategory.bind(this);
+    this.state = { title: '', category: 'Action' };
+    this.handleChange = this.handleChange.bind(this);
     this.handleAddBook = this.handleAddBook.bind(this);
   }
```

Comment:
:bulb:  It is redundant to use `bind()` with arrow functions. __Arrow functions establish "this" based on the scope the Arrow function is defined within.__ ([Read more](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#call_apply_and_bind))